.github/workflows/test-windows-fix.yaml

```diff
@@ -0,0 +1,71 @@
+# Built from:
+# https://docs.github.com/en/actions/guides/building-and-testing-python
+# https://github.com/snok/install-poetry#workflows-and-tips
+
+name: Build and test linkml
+
+on:
+  push:
+    branches:
+      - fix_docgen
```

Comment:
is this branch sticking around?

Reply:
No, @ialarmedalien, it will not.  The changes to docgen to add the truncate flag, broke mermaidclassdiagramgen and the tests in main, and I am using this to fail fast as I try to fix spacing, etc. in the windows environment (as replicating windows OS locally for me is not great).